modules/local/ensemblvep/main.nf

```diff
@@ -0,0 +1,52 @@
+process ENSEMBLVEP {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::ensembl-vep=104.3" : null)
```

Comment:
If at all possible we should have the same version of vep for conda as we have for the containers